src/main/kotlin/com/oneplanet/dto/request/ClimatiqEstimateRequest.kt

```diff
@@ -0,0 +1,6 @@
+package com.oneplanet.dto.request
+
+import io.micronaut.core.annotation.Introspected
+
+@Introspected
+data class ClimatiqEstimateRequest(val emission_factor : EmissionFactorRequest, val parameters: ParametersRequest)
```

Comment:
One field per line would be better (see the comment in `EmissionFactorResponse`)

Reply:
Also, avoid naming fields using `_`, instead use something like Jackson to rename the field when you need to send the request.